src/org/jitsi/util/RTPUtils.java

```diff
@@ -69,6 +71,27 @@ public static int writeInt(byte[] buf, int off, int data)
     }
 
     /**
+     * Writes the least significant 24 bits from the given integer into the
+     * given byte array at the given offset.
+     * @param buf the buffer into which to write.
+     * @param off the offset at which to write.
+     * @param data the integer to write.
+     * @return 3
+     */
+    public static int writeUint24(byte[] buf, int off, int data)
```

Comment:
feels like it belongs somewhere more generic than 'RTPUtils' but not a big deal for now i guess